superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -78,7 +78,12 @@ export interface FiltersConfigFormProps {
   parentFilters: { id: string; title: string }[];
 }
 
-const FILTERS_WITHOUT_COLUMN = ['filter_timegrain', 'filter_timecolumn'];
+// TODO: Need to do with it something
+const FILTERS_WITHOUT_COLUMN = [
+  'filter_timegrain',
+  'filter_timecolumn',
+  'filter_groupby',
+];
```

Comment:
Yes, we really need to address this at some point. I suggest once we do dynamic controls we make sure we can use the same control panel for the filter config modal and Explore.